lib/cc/engine/rubocop.rb

```diff
@@ -117,6 +119,18 @@ def content_body(cop_name)
         )
         File.exist?(path) ? File.read(path) : nil
       end
+
+      def cops
+        if File.exists?("Gemfile.lock")
+          detector = GemDetector.new(File.read("Gemfile.lock"))
```

Comment:
WDYT about letting `GemDetector` check existence and read the file itself? Then this method could be a little more squint-friendly:

```rb
detector = GemDetector.new("Gemfile.lock")

if detector.gem?("railties")
  RuboCop::Cop::Cop.all
else
  RuboCop::Cop::Cop.non_rails
end
```